frontend/src/pages/UserDashboard.jsx

```
@@ -0,0 +1,664 @@
+    }
+  };
+
+  const filteredProperties = properties.filter(property => 
+    property.name.toLowerCase().includes(searchQuery.toLowerCase()) ||
+    property.location?.toLowerCase().includes(searchQuery.toLowerCase())
+  );
+
+  const sortedProperties = [...filteredProperties].sort((a, b) => {
+    if (sortOption === 'name-asc') return a.name.localeCompare(b.name);
+    if (sortOption === 'name-desc') return b.name.localeCompare(a.name);
+    if (sortOption === 'price-asc') return parseFloat(a.price) - parseFloat(b.price);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)